sound/soc/sof/intel/hda.h

```diff
@@ -559,6 +559,7 @@ struct sof_intel_hda_stream {
 	struct sof_intel_stream sof_intel_stream;
 	int host_reserved; /* reserve host DMA channel */
 	u32 flags;
+	struct completion ioc;
```

Comment:
this needs completion.h now

Reply:
indeed, thanks for the attention to details @lyakh !